src/components/application_manager/rpc_plugins/vehicle_info_plugin/src/vehicle_info_pending_resumption_handler.cc

```diff
@@ -223,6 +223,14 @@ void VehicleInfoPendingResumptionHandler::HandleOnEvent(
   custom_vehicle_data_manager_.CreateMobileMessageParams(converted_msg_params);
   response_message[strings::msg_params] = converted_msg_params;
 
+  for (const std::string& param : converted_msg_params.enumerate()) {
+    if (param == strings::cluster_modes) {
+      response_message[strings::msg_params][strings::cluster_mode_status] =
```

Comment:
Should we do the assignments in a loop at each iteration ?

Reply:
no, only one change for clusterModes

Reply:
@VadymLuchko Please look at aee10406ba721ae01473a7bdeaa0febaaef6ef28